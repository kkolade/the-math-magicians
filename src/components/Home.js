import React from 'react';
import './home.css';

const Home = () => (
  <div className="Home-container">
    <img
      // src="https://media.giphy.com/media/NKLdcqhwo2f8A/giphy.gif"
      src="https://media.giphy.com/media/xUA7b6oaRIgzmAKpUY/giphy.gif"
      alt="professor doing math on a whiteboard"
      className="Home-hero-img"
    />
    <article>
      <h2 className="Home-headline">Welcome to our page!</h2>
      <p className="Home-text-content">
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Est optio
        inventore iste laudantium voluptatem earum? Itaque eligendi molestiae
        excepturi et vitae veniam quod consequatur quasi. Et repudiandae
        voluptate rem consequuntur adipisci maiores vitae nulla eum eaque? Illo
        quod porro veniam assumenda corrupti ipsa? Voluptate aspernatur unde vel
        sunt quidem incidunt ducimus repellat animi corrupti ab laudantium
        cupiditate optio obcaecati excepturi architecto ipsam necessitatibus
        natus, quia odio nisi veritatis minima. Ab officia, iste eius modi optio
        voluptas temporibus eum voluptates. Maxime illum aut consectetur
        officiis voluptatum eligendi quam quisquam odit, in numquam, quas error
        voluptas dolore. Fugit maxime est, sit illo magni labore veniam,
        deserunt neque culpa molestias, aut voluptatem debitis facilis commodi
        sint.
      </p>
      <p className="Home-text-content">
        Maxime animi voluptatum id, vitae aperiam culpa sint tenetur dolores
        laboriosam impedit quod nostrum quaerat similique quibusdam sunt
        voluptate ratione aut aliquam? Esse explicabo dicta cupiditate id porro
        ipsum similique natus accusamus maiores, doloremque deleniti ex impedit
        ea nobis cum repudiandae error quasi ducimus dolorum architecto rerum
        modi consequuntur corrupti! Iure esse rem molestias corporis, velit
        accusantium odit illo aspernatur fugit alias reprehenderit corrupti
        vitae provident doloribus iusto! Nostrum quas facilis illum a dicta
        officiis facere in voluptatibus, dolorum eligendi et, fugit tenetur
        deleniti incidunt quis! Tenetur!
      </p>
    </article>
  </div>
);

export default Home;
