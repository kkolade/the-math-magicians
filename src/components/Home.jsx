import React from 'react';
import './home.css';

const Home = () => (
  <div className="Home-container">
    <h2 className="Home-headline">Welcome to our page!</h2>
    <p className="Home-text-content">
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Est optio
      inventore iste laudantium voluptatem earum? Itaque eligendi molestiae
      excepturi et vitae veniam quod consequatur quasi. Et repudiandae voluptate
      rem consequuntur adipisci maiores vitae nulla eum eaque? Illo quod porro
      veniam assumenda corrupti ipsa? Voluptate aspernatur unde vel sunt quidem
      incidunt ducimus repellat animi corrupti ab laudantium cupiditate optio
      obcaecati excepturi architecto ipsam necessitatibus natus, quia odio nisi
      veritatis minima. Ab officia, iste eius modi optio voluptas temporibus eum
      voluptates. Maxime illum aut consectetur officiis voluptatum eligendi quam
      quisquam odit, in numquam, quas error voluptas dolore. Fugit maxime est,
      sit illo magni labore veniam, deserunt neque culpa molestias, aut
      voluptatem debitis facilis commodi sint.
    </p>
    <p className="Home-text-content">
      Maxime animi voluptatum id, vitae aperiam culpa sint tenetur dolores
      laboriosam impedit quod nostrum quaerat similique quibusdam sunt voluptate
      ratione aut aliquam? Esse explicabo dicta cupiditate id porro ipsum
      similique natus accusamus maiores, doloremque deleniti ex impedit ea nobis
      cum repudiandae error quasi ducimus dolorum architecto rerum modi
      consequuntur corrupti! Iure esse rem molestias corporis, velit accusantium
      odit illo aspernatur fugit alias reprehenderit corrupti vitae provident
      doloribus iusto! Nostrum quas facilis illum a dicta officiis facere in
      voluptatibus, dolorum eligendi et, fugit tenetur deleniti incidunt quis!
      Tenetur!
    </p>
  </div>
);

export default Home;
