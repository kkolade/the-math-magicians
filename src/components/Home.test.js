import React from 'react';
import { render } from '@testing-library/react';
import Home from './Home';

describe('Home component', () => {
  it('renders correctly', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });

  it('contains a headline', () => {
    const { getByText } = render(<Home />);
    expect(getByText('Welcome to our page!')).toBeInTheDocument();
  });
});
