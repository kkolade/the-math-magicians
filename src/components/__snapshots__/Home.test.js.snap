// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<div>
  <div
    class="Home-container"
  >
    <h2
      class="Home-headline"
    >
      Welcome to our page!
    </h2>
    <p
      class="Home-text-content"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Est optio inventore iste laudantium voluptatem earum? Itaque eligendi molestiae excepturi et vitae veniam quod consequatur quasi. Et repudiandae voluptate rem consequuntur adipisci maiores vitae nulla eum eaque? Illo quod porro veniam assumenda corrupti ipsa? Voluptate aspernatur unde vel sunt quidem incidunt ducimus repellat animi corrupti ab laudantium cupiditate optio obcaecati excepturi architecto ipsam necessitatibus natus, quia odio nisi veritatis minima. Ab officia, iste eius modi optio voluptas temporibus eum voluptates. Maxime illum aut consectetur officiis voluptatum eligendi quam quisquam odit, in numquam, quas error voluptas dolore. Fugit maxime est, sit illo magni labore veniam, deserunt neque culpa molestias, aut voluptatem debitis facilis commodi sint.
    </p>
    <p
      class="Home-text-content"
    >
      Maxime animi voluptatum id, vitae aperiam culpa sint tenetur dolores laboriosam impedit quod nostrum quaerat similique quibusdam sunt voluptate ratione aut aliquam? Esse explicabo dicta cupiditate id porro ipsum similique natus accusamus maiores, doloremque deleniti ex impedit ea nobis cum repudiandae error quasi ducimus dolorum architecto rerum modi consequuntur corrupti! Iure esse rem molestias corporis, velit accusantium odit illo aspernatur fugit alias reprehenderit corrupti vitae provident doloribus iusto! Nostrum quas facilis illum a dicta officiis facere in voluptatibus, dolorum eligendi et, fugit tenetur deleniti incidunt quis! Tenetur!
    </p>
  </div>
</div>
`;
